src/test/java/io/cryostat/core/reports/InterruptibleReportGeneratorTest.java

```diff
@@ -165,6 +167,50 @@ void shouldProduceEmptyReport() throws Exception {
         }
     }
 
+    @Test
+    void shouldProduceScoreMap() throws Exception {
+        try (InputStream is = new FileInputStream(getJfrFile())) {
+            Future<Map<String, ScoreRule>> scoreMap =
+                    generator.generateScoreMapInterruptibly(is, rule -> true);
+
+            Map<String, ScoreRule> s = scoreMap.get();
+
+            for (var entry : s.entrySet()) {
```

Comment:
Probably worth adding an assertion before this loop to ensure that the `entrySet` isn't empty too